cloud-controller-manager/do/loadbalancers_test.go

```diff
@@ -3411,13 +3411,24 @@ func Test_GetLoadBalancer(t *testing.T) {
 }
 
 func Test_EnsureLoadBalancer(t *testing.T) {
+	getGetCertFnFn := func(doCertType string) func(ctx context.Context, certID string) (*godo.Certificate, *godo.Response, error) {
+		f := func(ctx context.Context, certID string) (*godo.Certificate, *godo.Response, error) {
+			return &godo.Certificate{
+				ID:   certID,
+				Type: doCertType,
+			}, nil, nil
+		}
+		return f
+	}
+	defaultGetCertFn := getGetCertFnFn(certTypeCustom)
 	testcases := []struct {
 		name              string
 		droplets          []godo.Droplet
 		getFn             func(context.Context, string) (*godo.LoadBalancer, *godo.Response, error)
 		listFn            func(context.Context, *godo.ListOptions) ([]godo.LoadBalancer, *godo.Response, error)
 		createFn          func(context.Context, *godo.LoadBalancerRequest) (*godo.LoadBalancer, *godo.Response, error)
 		updateFn          func(ctx context.Context, lbID string, lbr *godo.LoadBalancerRequest) (*godo.LoadBalancer, *godo.Response, error)
+		getCertFn         func(context.Context, string) (*godo.Certificate, *godo.Response, error)
```

Comment:
We never seem to set this in any test case?

Reply:
This is only intended to keep the existing tests from breaking with the addition of `kvCertService` to the `newFakeClient` signature.